### DISCUSSION TITLE
Junior Certificate - exam question about Properties

### DISCUSSION THREAD

**Discussion by bernardpolman** [pimcore/pimcore#17032] [Q&A / Support] [answered]
In chapter "Working with Documents" one of the questions is:
**"Which statements concerning properties are correct?"**

**"Properties can be inherited via content master document."** - in the video it clearly states that children documents inherit properties from main documents, which is why I marked this as **true**.

**"The language of individual documents can be defined with a system property."** - I left this checkbox empty, i.e. **false** because there is no such predefined property, but rather a Language property in the right part next to properties window

**"Custom Properties need to be considered in custom code (business logic or template) to have any effect on the application."** - I marked this **true**, because logically, you need to handle custom properties yourself, since you added them....?

**"Properties can be used to influence navigation building."** - I also marked this **true** because per video, it is stated that navigation can be manipulated via properties, eg. showing or hiding left nav section.

What am I misunderstanding here?

**Comment by markus-moser** [comment, marked as answer]
Yes, properties can be inherited to children documents, but the content master document is something different. Take a look here https://pimcore.com/docs/platform/2023.3/Pimcore/Documents/Inheritance/ ;)

**Comment by ArminWindlin** [comment]
The language property in the right part is a 'system property'. So the language of individual documents **can** be defined with a system property.
